src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -0,0 +1,192 @@
+import { Card, Chip } from '@mui/material';
+import { capitalize } from 'lodash-es';
+import React, { useCallback, useMemo, useState } from 'react';
+import DeleteMutationButton from '@/modules/dataFetching/components/DeleteMutationButton';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import DynamicView from '@/modules/form/components/viewable/DynamicView';
+import { useStaticFormDialog } from '@/modules/form/hooks/useStaticFormDialog';
+import { FormValues } from '@/modules/form/types';
+import { getItemMap } from '@/modules/form/util/formUtil';
+import {
+  customDataElementValueForKey,
+  getCustomDataElementColumns,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import { cache } from '@/providers/apolloClient';
+import {
+  DeleteExternalFormSubmissionDocument,
+  DeleteExternalFormSubmissionMutation,
+  DeleteExternalFormSubmissionMutationVariables,
+  ExternalFormSubmissionFieldsFragment,
+  ExternalFormSubmissionFilterOptions,
+  ExternalFormSubmissionInput,
+  ExternalFormSubmissionStatus,
+  GetProjectExternalFormSubmissionsDocument,
+  GetProjectExternalFormSubmissionsQuery,
+  GetProjectExternalFormSubmissionsQueryVariables,
+  StaticFormRole,
+  UpdateExternalFormSubmissionDocument,
+  UpdateExternalFormSubmissionMutation,
+  UpdateExternalFormSubmissionMutationVariables,
+} from '@/types/gqlTypes';
+
+export type ExternalFormSubmissionFields = NonNullable<
+  GetProjectExternalFormSubmissionsQuery['project']
+>['externalFormSubmissions']['nodes'][number];
+
+const ProjectExternalSubmissionsTable = ({
+  projectId,
+  formDefinitionIdentifier,
+}: {
+  projectId: string;
+  formDefinitionIdentifier: string;
+}) => {
+  const getColumnDefs = useCallback((rows: ExternalFormSubmissionFields[]) => {
+    const customColumns = getCustomDataElementColumns(rows);
+    return [
+      {
+        header: 'Status',
+        linkTreatment: false,
+        render: (s: ExternalFormSubmissionFieldsFragment) => (
+          <Chip
+            label={capitalize(s.status)}
```

Comment:
maybe these chips could be a little less heavy on the page
```suggestion
            variant='outlined'
            size='small'```

Reply:
Thanks @ttoomey , the `small` size is indeed closer to the mocks so thank you for pointing that out! I also added the `outlined` variant for reviewed submissions. And I noticed the color of the chip for reviewed submissions should be `text.secondary`, so I've added it here.

I added it as `sx` after spending a little time trying to get the Chip to accept `text.secondary` as a color prop, using `ChipPropsColorOverrides`. I ran into issues with this -- similar to some of the problems described [here](https://stackoverflow.com/questions/69481467/mui-error-cannot-read-properties-of-undefined-reading-drawer) and in some of the comments [here](https://stackoverflow.com/questions/64932595/how-do-i-extend-the-color-palette-from-mui-with-typescript) -- basically Chip color properties can't be quoted and they require a contrast color to be defined. Anyway, I still think this is possible but decided not to proceed, because I think we lose some of the benefits if it has to be `textSecondary` instead of `text.secondary` because of that quote issue. Let me know what you think.

Here's how it looks for now:
<img width="352" alt="Screenshot 2024-03-14 at 1 54 49 PM" src="https://github.com/greenriver/hmis-frontend/assets/16002775/097f3d7b-ed1b-4532-b448-82d2b196772a">

Reply:
Thanks Martha, that looks good! In the future maybe we can extract this into a common component that wraps Mui's Chip (CommonChip or something).